media-gfx/imv/files/imv-4.5.0-libnsgif-1.0.0.patch

```diff
@@ -0,0 +1,231 @@
+https://bugs.gentoo.org/922496
+https://todo.sr.ht/~exec64/imv/55
+https://lists.sr.ht/~exec64/imv-devel/%3C20241113012702.30521-2-reallyjohnreed%40gmail.com%3E
+
+From: John Reed <reallyjohnreed@gmail.com>
+To: ~exec64/imv-devel@lists.sr.ht
+Cc: John Reed <reallyjohnreed+git@gmail.com>
+Subject: [PATCH imv] update libnsgif backend code to match with libnsgif v1.0.0 and re-enable gif viewing
+Date: Tue, 12 Nov 2024 20:27:03 -0500
+Message-ID: <20241113012702.30521-2-reallyjohnreed@gmail.com>
+X-Mailer: git-send-email 2.46.1
+MIME-Version: 1.0
+Content-Transfer-Encoding: 8bit
+
+From: John Reed <reallyjohnreed+git@gmail.com>
+
+Fixes: https://todo.sr.ht/~exec64/imv/55
+---
+ meson.build            |  2 +-
+ src/backend_libnsgif.c | 92 ++++++++++++++++++++++++++----------------
+ 2 files changed, 58 insertions(+), 36 deletions(-)
+
+--- a/meson.build
++++ b/meson.build
+@@ -125,7 +125,7 @@ foreach backend : [
+   ['libpng', 'dependency', 'libpng', []],
+   ['libjpeg', 'dependency', 'libturbojpeg', []],
+   ['librsvg', 'dependency', 'librsvg-2.0', '>= 2.44'],
+-  ['libnsgif', 'dependency', 'libnsgif', '< 1.0.0'],
++  ['libnsgif', 'dependency', 'libnsgif', '1.0.0'],

```

Comment:
This will actually break with a future libnsgif-1.0.1 or so, as far as I can tell this requests the exact 1.0.0 version rather than `>= 1.0.0` like librsvg does above.